run.py

```diff
@@ -962,6 +1012,8 @@ def engine_manifest():
         "--allow_origin", nargs="*", help="許可するオリジンを指定します。複数指定する場合は、直後にスペースで区切って追加できます。"
     )
 
+    parser.add_argument("--setting_file", type=Path, help="設定ファイルを手動で設定います。")
```

Comment:
誤字かも？
あと「手動で」は意図が伝わりにくそうなので、こんな感じでどうでしょう 👀
```suggestion
    parser.add_argument("--setting_file", type=Path, help="設定ファイルを指定できます。")
```